test/functional/wallet_labels.py

```diff
@@ -71,6 +71,10 @@ def run_test(self):
         node = self.nodes[0]
         assert_equal(len(node.listunspent()), 0)
 
+        self.log.info("Checking listlabels' invalid parameters")
+        assert_raises_rpc_error(-8, "Invalid 'purpose' argument, must be one of 'send', or 'receive'.", node.listlabels, "notavalidpurpose")
+        assert_raises_rpc_error(-8, "Invalid 'purpose' argument, must be one of 'send', or 'receive'.", node.listlabels, "unknown")
```

Comment:
In commit "wallet: Replace use of purpose strings with an enum" (db521d3e89b23046f73410a875bb9660178e3842)

I think the PR might benefit from a release note, or at least a summary of behavior changes in the PR description like, 'Wallet loading code will now issue warnings if addresses have unrecognized purpose strings other than "send" "receive" and "refund" and `listlabels` RPC will raise an error if an unrecognized purpose is requested.'

Reply:
Done